### PR TITLE
Added note about supported clients for background blur features to meeting-policies-in-teams.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/6054.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/meeting-policies-in-teams.md [hunk 1]
```diff
@@ -539,9 +539,12 @@ To specify whether users can customize their video background in a meeting, set
 |**BlurandDefaultBackgrounds**     |User has the option to blur their video background or choose from the default set of images to use as their background. |
 |**AllFilters**     |Use has the option to blur their video background, choose from the default set of images, or upload custom images to use as their background. |
```
Comment (JohanFreelancer9) [inline Teams/meeting-policies-in-teams.md:540]:
```suggestion
|**AllFilters**     |User has the option to blur their video background, choose from the default set of images, or upload custom images to use as their background. |
```

File: Teams/meeting-policies-in-teams.md [hunk 2]
```diff
@@ -539,9 +539,12 @@ To specify whether users can customize their video background in a meeting, set
...
 
-> [!NOTE]
+> [!IMPORTANT]
 > Images uploaded by users aren't screened by Teams. When you use the **AllFilters** setting, you should have internal organization policies to prevent users from uploading offensive or inappropriate images, or images your organization don't have rights to use for Teams meeting backgrounds.
 
+> [!NOTE]
+> These features are not available for all Teams clients. For more information, see _Video and backgrounds_ title in [Meetings and live events](https://support.microsoft.com/office/meetings-and-live-events-5c3e0646-dc37-45ad-84a4-1666fac62d4e).
```
Comment (JohanFreelancer9) [inline Teams/meeting-policies-in-teams.md:546]:
```suggestion
> These features are not available for all Teams clients. For more information, see the _Video and backgrounds_ title in [Meetings and live events](https://support.microsoft.com/office/meetings-and-live-events-5c3e0646-dc37-45ad-84a4-1666fac62d4e).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit and one additional correction to Line 540, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**